spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/ZeroPadding3D.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl.nn.Padding
+import com.intel.analytics.bigdl.nn.abstractnn._
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.nn.{Sequential => TSequential}
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+
+import scala.reflect.ClassTag
+
+/**
+  * Zero-padding layer for 3D data (spatial or spatio-temporal).
+  *
+  * @param padding Int array of length 3. How many zeros to add at the beginning and end of the 3 padding dimensions
+  *                (axis 3, 4 and 5). Currently only symmetric padding is supported. Default is (1, 1, 1).
```

Comment:
Remove (axis 3, 4, 5)
Change to `Symmetric padding will be applied to each dimension.`